### PR TITLE
Project-movies Week 12 Linnea Frisk Jenny Fiskaare

### PR DESCRIPTION
**Pull request by Neaa99**
Link to my deployed project:
https://project-movies-linnea-and-jenny.netlify.app/

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://project-movies-linnea-and-jenny.netlify.app/
```
Comment (CamillaHallberg) [inline README.md:8]: Your page looks absolutely on point to the design provided for this week. I especially like the hover effect to darken each movie and how it transitions. Really great job accomplishing the design this week! :)

File: code/public/index.html [hunk 2]
```diff
@@ -4,31 +4,18 @@
...
 
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>Technigo React App</title>
+    <meta property="og:title" content="Popular movies by Jenny Fiskaare & Linnea Frisk" />
+    <meta property="og:description" content="A site where you can scroll through the latest popular movies"/>
+    <meta property="og:image" content="https://i.postimg.cc/y8bBnsfR/Sk-rmklippmovie.png"/>
+    <meta property="og:url" content="https://project-movies-linnea-and-jenny.netlify.app/"/>
```
Comment (CamillaHallberg) [inline code/public/index.html:11]: Nice that you used this property to make content shareable! :) I tried it out by adding your netlify link to create a post on LinkedIn and the preview looks nice! :) Do you mind explaining the image, how did you create this? Is it basically a screen shot of the netlify page? I would like to implement this myself on future projects so thanks for the inspiration :)

File: code/src/App.js [hunk 3]
```diff
@@ -1,9 +1,20 @@
...
 
 export const App = () => {
   return (
     <div>
-      Find me in src/app.js!
+      <BrowserRouter>
+        <Routes>
+          <Route path='/' element={<Movie />}></Route>
+          <Route path='/movie/:id' element={<Details />}></Route>
+
```
Comment (CamillaHallberg) [inline code/src/App.js:15]: You can remove this little empty line :)

File: code/src/components/BackIcon.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+export const BackIcon = () => {
+    return(
+        <div className="icon-box">
+            <svg className="icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 30 30">
+                <path d="M27 14.5C27 7.596441 21.4035594 2 14.5 2S2 7.596441 2 14.5 7.5964406 27 14.5 27 27 21.403559 27 14.5zm-19.3388348-.353553l7.4852814-7.485282c.1952622-.195262.5118446-.195262.7071068 0l2.1213203 2.121321c.1952622.195262.1952622.511844 0 .707106L12.9644661 14.5l5.0104076 5.010408c.1952622.195262.1952622.511844 0 .707106l-2.1213203 2.121321c-.1952622.195262-.5118446.195262-.7071068 0l-7.4852814-7.485282c-.19799-.19799-.197989-.509117 0-.707106z" fill="#fff" fillRule="evenodd" />
+            </svg>
```
Comment (CamillaHallberg) [inline code/src/components/BackIcon.js:8]: Love the back arrow icon :)

File: code/src/components/Details.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+const Details = () => {
+
+    const [details, setDetails] = useState([])
+    const { id } = useParams()
+
+    useEffect(() => {
+        fetch(`https://api.themoviedb.org/3/movie/${id}?api_key=${'2ef0cecd459c1c1851bbe1d699a604a0'}&language=en-US`)
+            .then(res => res.json())
+            .then(data => {
+                console.log(data)
```
Comment (CamillaHallberg) [inline code/src/components/Details.js:15]: You can remove console log here.

File: code/src/components/Details.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    return (
+            <div className="details-body"> 
+                <img className="movie-poster" src={`https://image.tmdb.org/t/p/w1280${details.backdrop_path}`} alt={details.title}/>
+                
+                <NavLink class="backLink" to='/'>
+                   <BackIcon />
+                </NavLink>
+                
+                <div className="movie-box">
+                    <img className="movie-image" src={`https://image.tmdb.org/t/p/w342${details.poster_path}`} alt={details.title} tabindex="0"/>
```
Comment (CamillaHallberg) [inline code/src/components/Details.js:29]: I like that you thought of keyboard accessibility by using the tabindex attribute, tried it and it works :)

File: code/src/components/Movie.js [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+const Movie = () => {
+
+    const [movies, setMovies] = useState([])
+
+    useEffect(() => {
+        fetch(`https://api.themoviedb.org/3/movie/popular?api_key=${'2ef0cecd459c1c1851bbe1d699a604a0'}&language=en-US&page=1`)
+            .then(res => res.json())
+            .then(data => {
+                console.log(data)
```
Comment (CamillaHallberg) [inline code/src/components/Movie.js:12]: You can remove this console log too :)

File: code/public/index.html (hunk 2 above)
Comment (JustineZwiazek) [reply code/public/index.html:11]: This is so interesting, learned something new!

File: code/src/components/Movie.js [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+    return (
+        <div className="homePage">
+
+        {movies.map((movie) => (
+            <Link
+                to={`/movie/${movie.id}`} 
+                key={movie.id}
+                tabindex="0">
+                
+            <img src={`https://image.tmdb.org/t/p/w342${movie.poster_path}`} alt={movie.title}/>
```
Comment (JustineZwiazek) [inline code/src/components/Movie.js:26]: Really well done for doing the hover effect without using an overlay here. I am going to have a look at the css to see how you did that ⭐

File: code/src/index.css [hunk 9]
```diff
@@ -1,13 +1,159 @@
...
+  left: 0;
+  bottom: 0;
+  background: rgba(0, 0, 0, 0.75);
+  padding: 20px;
+  opacity: 0;
+}
+
+.homePage a:hover .details {
+  opacity: 1;
+  transition: ease-in 0.4s;
```
Comment (JustineZwiazek) [inline code/src/index.css:47]: Really liked the slow transition here.

**Review by JustineZwiazek — COMMENTED** [review]
Really nice job with the project, the site looks like the sample design and you also included the smaller details such as the release date on the main page / score on the details page. Prompted by your project I am going to read up on meta property :)

**Pull request closed without merging**